pages/implementations/earl-reports.md

```diff
@@ -71,13 +71,44 @@ The following properties are required for each Assertion:
 			"assertions": [
 				{
 					"@type": "Assertion",
-					"test": { "title": "My Tool's rule title" },
-					"result": { "outcome": "earl:passed" }
+					"result": { "outcome": "earl:passed" },
+					"test": { ... },
 				}
 			]
 		}
 	]
 }
 ```
 
+### Test Criterion
+
+For each result, we'll need to know which rule or procedure in the tool reported it, and which WCAG success criteria were failed. To do so, on the `test` property add an object. This must include:
+
+- `title`: A string, the uniquely identifies what rule or procedure in the tool found the result.
+- `isPartOf`: An array of strings, listing the WCAG success criteria that fail when the rule or procedure fails. These must use the IDs of the WCAG success criteria, prefaced with `WCAG2:`. For example `WCAG2:name-role-value`.
+
+```json
+{
+	"@context": "https://act-rules.github.io/earl-context.json",
+	"@graph": [
+		{
+			"@type": "TestSubject",
+			"source": "https://act-rules.github.io/testcases/a1b64e/6c3ac31577c3cb2d968fc26c4075dd533b5513fc.html",
+			"assertions": [
+				{
+					"@type": "Assertion",
+					"result": { "outcome": "earl:passed" },
+					"test": {
+						"title": "image-button-has-name",
+						"isPartOf": ["WCAG2:non-text-content", "WCAG2:name-role-value"]
+					}
+				}
+			]
+		}
+	]
+}
+```
+
+**Note**: The IDs of WCAG success criteria can be found in [sc-urls.json](https://github.com/act-rules/act-tools/blob/main/src/data/sc-urls.json), on the `scId` property. These are the IDs introduced in WCAG 2.1, and continued in WCAG 2.2. Use these IDs, even when reporting for WCAG 2.0.
+
 **Advanced**: All required properties map to [EARL](http://www.w3.org/ns/earl#), except for `title` and `source` which are properties of [Dublin Core](http://purl.org/dc/terms/). Property names can be anything, as long as they can be expanded to the correct URL.
```

Comment:
I'm guessing `isPartOf` isn't part of `EARL`? Does this statement need an update?